contracts/contracts/vault/VaultAdmin.sol

```diff
@@ -343,6 +343,45 @@ contract VaultAdmin is VaultStorage {
         emit AssetSupported(_asset);
     }
 
+    /**
+     * @notice Remove a supported asset from the Vault
+     * @param _asset Address of asset
+     */
+    function removeAsset(address _asset) external onlyGovernor {
+        require(assets[_asset].isSupported, "Asset not supported");
+        require(
+            IVault(address(this)).checkBalance(_asset) <= 1e13,
```

Comment:
why an external call and not an internal `_checkBalance(_asset)`?

Reply:
`checkBalance` is in `VaultCore`, not `VaultAdmin`

Reply:
Ah, yes, that makes sense.